### ISSUE TITLE
Issue with training- only produces 4k files that don't seem to work

### ISSUE THREAD

**Issue by juan9999** [AUTOMATIC1111/stable-diffusion-webui-aesthetic-gradients#19]
I installed the extension via extensions in 1111.

I resized 10 images to  512x512, put them in a folder (I know it finds them as I mistyped the path and got error messages), I then create the embedding and get Done generating embedding message and the embedding is placed in the correct directory. It's done generating super quick but it only generates a 4k .pt file. Tried it with two different training sets of 10 images.

On the txt2img screen, I select a 1.5 variation that I trained with a face, then under Clip Aesthetic I select the aesthetic that I trained, and have varied the steps from 10-max, and changed the weight from .1 to 1 but nothing happens. 

Any ideas?

**Comment by magejosh** [comment]
.pt files from embeddings are supposed to be that size generally speaking, at least as far as the ones you get from normal textual inversion trained embeddings, but i am just looking into this extension today. I'll report back after i've tried making some if i have any success.
